vortex-ipc/src/io/mod.rs

```diff
@@ -9,6 +10,7 @@ mod write;
 pub use futures::*;
 #[cfg(feature = "monoio")]
 pub use monoio::*;
+pub use object_store::*;
```

Comment:
conditional compile in case people don't want to build all of object_store? e.g. in tests